### PR TITLE
LPS-182721 Replace usages of old liferay-ui:logo-selector taglib

### PR DESCRIPTION
**Pull request by sandrodw3**
## Motivation
This PR replaces usages of `liferay-ui:logo-selector` taglib with the new `liferay-frontend:logo-selector` one.

**Only LPS-182721 commits need to be reviewed.**

## How to reproduce
I'm leaving the steps to reach the views:
### `account-admin-web`
- Apps Menu > Control Panel > Accounts > Create an account
- Apps Menu > Control Panel > Account Users > Create an account user

**Note**: For these views, I have changed the view from 2 columns to 1 so it looks better, according to our designers.

### `users-admin-web`
- Apps Menu > Control Panel > Users and Organizations > Create/edit a user
- Apps Menu > Control Panel > Users and Organizations > Create/edit an organization

**Note**: For these views, I have changed the view from 2 columns to 1 so it looks better, according to our designers.

### `oauth2-provider-web`
- Apps Menu > Control Panel > OAuth 2 Administration > Edit one of them

### `portal-settings-web`
- Apps Menu > Control Panel > Instance Settings > Instance Configuration > Appearance

### CODE REVIEW

**Comment by sandrodw3** [comment]
ci:test:sf

**Comment by ealonso** [comment]
ci:test:relevant

**Comment by sandrodw3** [comment]
ci:stop

**Comment by sandrodw3** [comment]
ci:test:sf

**Comment by sandrodw3** [comment]
ci:test:relevant

File: modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_entries_admin/account_entry/display_data.jsp [hunk 1]
```diff
@@ -27,61 +27,46 @@ String[] types = GetterUtil.getStringValues(request.getAttribute(AccountWebKeys.
...
-
-		<clay:col
-			md="5"
-		>
-			<div align="middle">
-				<label class="control-label"></label>
-
-				<liferay-ui:logo-selector
-					currentLogoURL="<%= accountEntryDisplay.getLogoURL() %>"
-					defaultLogo="<%= accountEntryDisplay.getLogoId() == 0 %>"
```
Comment (claraizquierdo) [inline modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_entries_admin/account_entry/display_data.jsp:78]: In the old taglib we could pass this boolean parameter default logo. Don't we need something similar in the new taglib?

File: modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_entries_admin/account_entry/display_data.jsp [hunk 2]
```diff
@@ -27,61 +27,46 @@ String[] types = GetterUtil.getStringValues(request.getAttribute(AccountWebKeys.
...
-					defaultLogoURL="<%= accountEntryDisplay.getDefaultLogoURL() %>"
-					tempImageFileName="<%= String.valueOf(accountEntryDisplay.getAccountEntryId()) %>"
```
Comment (claraizquierdo) [inline modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_entries_admin/account_entry/display_data.jsp:80]: The same with `tempImageFileName`, don't we need something similar in the new taglib?

File: modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/organization/details.jsp [hunk 3]
```diff
@@ -37,93 +37,64 @@ if (organization != null) {
...
-		<div align="middle">
-			<c:choose>
-				<c:when test="<%= organization != null %>">
-					<label class="control-label"></label>
-
-					<liferay-ui:logo-selector
-						currentLogoURL="<%= organization.getLogoURL() %>"
-						defaultLogo="<%= organization.getLogoId() == 0 %>"
-						defaultLogoURL='<%= themeDisplay.getPathImage() + "/organization_logo?img_id=0" %>'
-						logoDisplaySelector=".organization-logo"
```
Comment (claraizquierdo) [inline modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/organization/details.jsp:110]: The `logoDisplaySelector` is not passed to the new taglib. Is it not needed?

File: modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_entries_admin/account_entry/display_data.jsp (hunk 2 above)
Comment (sandrodw3) [reply modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_entries_admin/account_entry/display_data.jsp:80]: As far as we could see, this one was not actually being used for nothing 🤷‍♂️

File: modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/organization/details.jsp (hunk 3 above)
Comment (sandrodw3) [reply modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/organization/details.jsp:110]: This was used to add that class to the image preview, but as far as I could see, it was being used for adding some margins/paddings and with the new design it is not needed. Now that you say, we can probably remove that CSS that is not needed anymore 🤔

Comment (claraizquierdo) [reply modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/organization/details.jsp:110]: Understood 👍 . As you say, if it is not used we should remove the CSS.

**Review by claraizquierdo — COMMENTED** [review]
Hi @sandrodw3 , the code seems to be working fine, but I have seen some visual changes that I would like to confirm with you:

- In Instance configuration now I'm allowed to clear the default logo and that option was not present before:
   
Captura de pantalla 2023-04-27 a las 17 16 14

- The name of the logo is in an input that looks editable but it is not. It looks a little bit confusing to me:
Captura de pantalla 2023-04-27 a las 17 10 53

- In OAuth administration we have deleted a title and I don't know if it is what we wanted:
Captura de pantalla 2023-04-27 a las 17 14 13

- Changes from 2 columns to one column: I would confirm that visual changes with the responsable teams so that they are aware of the changes.

**Comment by sandrodw3** [comment]
@claraizquierdo I applied the changes we accorded, I left more details in our slack chat 😄 please tell me when you can take a look, thanks!

**Comment by claraizquierdo** [comment]
reviewing it again

**Review by claraizquierdo — APPROVED** [review]
Hi @sandrodw3! Thank you for your changes. Everything seems to be working fine. From my previous comments the only thing I would continue saying is that I would confirm that visual changes with the responsable teams so that they are aware of the changes.

**Comment by ealonso** [comment]
see: https://github.com/liferay-echo/liferay-portal/pull/11938

**Pull request closed without merging**